clients/pkg/promtail/targets/cloudflare/target.go

```diff
@@ -28,6 +28,8 @@ import (
 // The minimun window size is 1 minute.
 const minDelay = time.Minute
 
+const cloudflareTooEarlyError = "too early: logs older than 168h0m0s are not available"
```

Comment:
I'm still hesitant about including specific details here, because it's too fragile. If CF updates their max lookback period from `168h0m0s` to `168h0m1s` (a silly example), this will break. Maybe we should just have this as:

```suggestion
const cloudflareTooEarlyError = "too early: logs older than \S+ are not available"
```

and use regex to match.

That should hopefully be a bit more resilient, given that the CF client is quite flimsy already in that it doesn't give us access to any other fields besides the iterator and the err AFAICS.

Reply:
Updated to regexp as requested.